### PR TITLE
Enhancing user's profile

### LINKED ISSUE #2375: Profile Enhancements
Please add the following changes to the gitcoin profiles

1. Add Bounties In Progress tab
2. We could borrow the 'Work Started', 'Work Submitted' etc. tags from the Funder Dashboard so that the activity entries are clearer (this would solve #2340 )
3. the pictures should show the repo logos, not the funder avatars

profile

### PR DESCRIPTION
**Pull request by nanspro**
##### Description

User's profile is more enhanced now due to presence of "Bounties In Progress" and more improvements to UI.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)

##### Affected core subsystem(s)
Front end

##### Testing
locally

##### Refers/Fixes
Fixes #2375 , #2340

### CODE REVIEW

**Comment by thelostone-mc** [comment]
@nanspro could you throw in screenshots / recording of the change ?

**Comment by nanspro** [comment]
screenshot from 2018-11-12 17-18-55

File: app/dashboard/templates/shared/profile_activities_bounty.html [hunk 1]
```diff
@@ -4,13 +4,18 @@
  {% elif activity.bounty.github_url %} href="{{ activity.bounty.github_url }}"
  {% endif %}>
   <div class="avatar-container col-1 justify-content-center special_tag hide_min_viewport">
-    <img class="avatar" src="{% url 'org_avatar' activity.bounty.bounty_owner_github_username %}?repo=https://github.com/{{ activity.bounty.bounty_owner_github_username }}">
+    {% if activity.bounty.funding_organisation %}
+      <img class="avatar" src="{% url 'org_avatar' activity.bounty.funding_organisation %}">
+    {% else %}
+    <img class="avatar" src="{% url 'org_avatar' activity.bounty.bounty_owner_github_username %}">
```
Comment (thelostone-mc) [inline app/dashboard/templates/shared/profile_activities_bounty.html]: Could we indent this by 1 ?

File: app/dashboard/views.py [hunk 2]
```diff
@@ -1142,6 +1142,8 @@ def profile(request, handle):
             profile = profile_helper(handle, current_user=request.user)
 
         context = profile.to_dict()
+        for x in context['activities']:
```
Comment (thelostone-mc) [inline app/dashboard/views.py]: Could we rename it something more meaningful ?

**Comment by thelostone-mc** [comment]
@nanspro code looks good to me! Could you address the minor changes ?

**Comment by nanspro** [comment]
@thelostone-mc  @owocki  ready to be merged!!!

**Pull request merged**